app/src/main/java/com/example/tasky/login/domain/LoginViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.example.tasky.login.domain
+
+import androidx.compose.ui.text.input.TextFieldValue
+
+class LoginViewModel {
+}
+
+data class LoginState(
+    val isValidEmail: Boolean,
+    val isLoading: Boolean,
+    val emailValue: TextFieldValue,
```

Comment:
Just a question

Why are you using TextfieldValue instead of a regular String?

Reply:
I thought that the viewModel could be the one who resolves the unpacking of the text from the TextFieldValue and also creating the TextFieldValue but giving it another look its easier that the state has a regular String